tests/issue-86/.laminas-ci.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
instead of `issue-86`, would it make sense to name this according to what it does?